driver/razeraccessory_driver.h

```diff
@@ -15,6 +15,7 @@
 #define USB_DEVICE_ID_RAZER_GOLIATHUS_CHROMA 0x0C01
 #define USB_DEVICE_ID_RAZER_GOLIATHUS_CHROMA_EXTENDED 0x0C02
 #define USB_DEVICE_ID_RAZER_FIREFLY_V2 0x0C04
+#define USB_DEVICE_ID_RAZER_FIREFLY_V2_PRO 0x0C08
```

Comment:
Keep this list sorted also, move it one line down

Reply:
i did it!